flang/lib/Lower/OpenMP/DataSharingProcessor.cpp

```diff
@@ -365,6 +409,9 @@ void DataSharingProcessor::collectDefaultSymbols() {
                        defaultSymbols);
     }
   }
+
+  for (auto *sym : defaultSymbols)
+    allPrivatizedSymbols.insert(sym);
```

Comment:
This should not be needed, as default symbols are collected through `collectSymbols()`.

Reply:
Indeed. Done.